### ISSUE TITLE
Monitorian fails on startup

### ISSUE THREAD

**Issue by emoacht** [emoacht/Monitorian#404]
### Discussed in https://github.com/emoacht/Monitorian/discussions/403

<div type='discussions-op-text'>

<sup>Originally posted by **ggammelgaard** January  3, 2023</sup>
Hi emoacht.
I love your little application here. Ufortunately it has stopped working for me 😥

When launching Monitorian a message box is prompted with the message "An unexpected problem occurred. Save exception log to Desktop?"

The content of the error log looks as follows:
```
[Date: 03/01/2023 07.24.44 Ver: 4.0.0.0]
System.ArgumentNullException: Value cannot be null.
Parameter name: iconPath HResult: -2147467261 ParamName: iconPath
   at ScreenFrame.NotifyIconContainer.ShowIcon(String iconPath, String iconText)
   at Monitorian.Core.AppControllerCore.<InitiateAsync>d__25.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Monitorian.AppController.<InitiateAsync>d__20.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Monitorian.App.<OnStartup>d__2.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)

[Date: 03/01/2023 07.24.48 Ver: 4.0.0.0]
System.ArgumentNullException: Value cannot be null.
Parameter name: iconPath HResult: -2147467261 ParamName: iconPath
   at ScreenFrame.NotifyIconContainer.ShowIcon(String iconPath, String iconText)
   at Monitorian.Core.AppControllerCore.<InitiateAsync>d__25.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Monitorian.AppController.<InitiateAsync>d__20.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Monitorian.App.<OnStartup>d__2.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.DispatcherOperation.InvokeImpl()
   at System.Windows.Threading.DispatcherOperation.InvokeInSecurityContext(Object state)
   at MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   at MS.Internal.CulturePreservingExecutionContext.Run(CulturePreservingExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Windows.Threading.DispatcherOperation.Invoke()
   at System.Windows.Threading.Dispatcher.ProcessQueue()
   at System.Windows.Threading.Dispatcher.WndProcHook(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
   at MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)
   at System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
   at System.Windows.Threading.Dispatcher.PushFrame(DispatcherFrame frame)
   at System.Windows.Application.RunDispatcher(Object ignore)
   at System.Windows.Application.RunInternal(Window window)
   at System.Windows.Application.Run(Window window)
   at Monitorian.App.Main()
```
</div>

**Comment by emoacht** [comment]
It's the same issue as #402.
What is your OS version?

**Comment by ggammelgaard** [comment]
OS Name	Microsoft Windows 10 Pro
Version	10.0.19044 Build 19044

**Comment by ggammelgaard** [comment]
I don't have the key SystemUsesLightTheme mentioned in #402 either.

**Comment by emoacht** [comment]
Thanks.
Then, what is your settings for Colors?
In the case of Custom, which you choose your default Windows mode (Light or Dark)?

**Comment by ggammelgaard** [comment]
I'm not sure i understand the question, but here is the other registry editor values:
![image](https://user-images.githubusercontent.com/17668756/210311524-be0ce0cb-5b1c-4ebf-8143-1c9bcef29d94.png)
And here is my color settings:
![image](https://user-images.githubusercontent.com/17668756/210311722-2e0e6de5-ff7e-44c7-aefe-5df5d9c10c87.png)

**Comment by emoacht** [comment]
Thanks. It is very helpful.

**Comment by MagicAndre1981** [comment]
this is same like my issue and I also only have those 2 keys shown in the picture.

**Comment by emoacht** [comment]
I checked freshly re-installed Windows 11 and it has `SystemUsesLightTheme` key while the theme is default ( light).

<img width="501" alt="regedit-personalize-win11-default" src="https://user-images.githubusercontent.com/7205690/210390439-e4d4abfb-d302-484d-8840-c50cb1b4b38b.png">

So it seems safe to conclude that if this key does not exists, the theme is dark.

**Comment by MagicAndre1981** [comment]
this key was added in 1903. before this version (so my 1809) taskbar was always dark and you can only select if the apps are light or dark.

**Comment by emoacht** [comment]
Tempolarily fixed by 81bd24ad7a9081fad722aa9c2cb2fc130c940259

**Comment by MagicAndre1981** [comment]
> Tempolarily fixed by [81bd24a](https://github.com/emoacht/Monitorian/commit/81bd24ad7a9081fad722aa9c2cb2fc130c940259)

works fine now on 1809

**Comment by ggammelgaard** [comment]
It works for me without error as well. Thank you very much!
